src/fsharp/NicePrint.fs

```diff
@@ -2507,3 +2507,11 @@ let minimalStringOfType denv ty =
     let denvMin = { denv with showImperativeTyparAnnotations=false; showConstraintTyparAnnotations=false }
     showL (PrintTypes.layoutTypeWithInfoAndPrec denvMin SimplifyTypes.typeSimplificationInfo0 2 ty)
 
+let prettyStringOfTy2 denv ty =
+    match ty with
+    | ty when isArrayTy denv.g ty -> "array"
```

Comment:
These strings all need to be localized, see the entries in FSComp.txt and the corresponding ways of fetching them out in code